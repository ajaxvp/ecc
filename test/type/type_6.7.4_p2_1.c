/* ISO: 6.7.4 (2); function specifier type constraint */

// valid usage of the specifier
inline int f(void) {}
// invalid usage of the specifier
inline char a;
