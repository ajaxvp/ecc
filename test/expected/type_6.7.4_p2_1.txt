ecc: error: [6:1] function specifiers may only be used in function types
